src/app/shared/ui/circular-button/circular-button.scss

```diff
@@ -0,0 +1,80 @@
+:host button{
+    --base-size: 40px;
+    --color1: #211918;
+    --color2: #231D18;
+    --color3: #AAA096;
+    --color4: #F1E3E3;
+    --color5: #D9D6D3;
+    --color6: #CAC7C5;
+    --color7: #B9B9B9;
+    --color8: #A4A4A4;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: none;
+    border-radius: 50%;
+    box-shadow: 0 0 0 1px var(--color1), 0 0 0 2px var(--color2), 0 0 0 3px var(--color3), 0 0 0 4px var(--color4), 0 0 0 5px var(--color5), 0 0 0 6px var(--color6), 0 0 0 7px var(--color7), 0 0 0 8px var(--color8);
+    background: linear-gradient(304.91deg, #323232 14.59%, #2C2120 82.16%);
+    transition: all .3s;
+
+    
+    
+    .icon{
```

Comment:
trochę za dużo tu tych linijek pustych

Reply:
Przy pisaniu często używałem komentarzy, przy kasowaniu przeoczyłem to. Done